magento/axp-connector
# Private Content Datalayer refactoring

## Description
This Pull Request does not introduce any functional changes. Multiple classes refactored, but the behavior should remain the same once merged.

## Scope
 - Extracted business logic from Helper to `Datalayer` model for Customer specific events.
 - Unified event format.
 - Updated "Order Placed" event to use Customer Data mechanism instead of cookies

## Manual testing
1. Add product to cart, verify that browser console has message form "Launch" with fired add to cart event.
2. Remove product from cart, verify that appropriate event is fired.
3. Place order on one page checkout, verify event.
4. Place order in Multishipping checkout, verify event.
